jobs/jobs.md

```diff
@@ -8,19 +8,19 @@ In the last MAD version a job processor was implemented. With this tool are you
 
 ### Job Types
 - Manual Jobs - start with MADmin
-- [Automatic Jobs](automatic_jobs)
+- [Automatic Jobs](automatic_jobs.md)
 
 ### Job Monitor
 
-see [Jobmonitor](job_monitor)
+see [Job Monitor](job_monitor.md)
 
 ### Discord Alert
 
-see [Discord Alert](discord_alert)
+see [Discord Alert](discord_alert.md)
 
 ### MADmin Endpoint
 
-see [MADmin_Endpoint](MADmin_Endpoint)
+see [MADmin_Endpoint](MADmin_Endpoint.md)
```

Comment:
The links are working perfectly fine atm

Reply:
@muckelba see here for broken links
https://github.com/Map-A-Droid/MAD-docs/blob/master/jobs/jobs.md

I know this is made to be viewed on readthedocs, but some may find it on Github 🤷‍♂